Lib/http/client.py

```diff
@@ -1171,6 +1171,12 @@ def putrequest(self, method, url, skip_host=False,
                         netloc_enc = netloc.encode("ascii")
                     except UnicodeEncodeError:
                         netloc_enc = netloc.encode("idna")
+
+                    # remove interface scope from IPv6 address
+                    # when used as Host header
+                    if netloc.find('%') >= 0:
```

Comment:
Why not: if "%" in netloc?

Reply:
just wanted to be aligned with the other conditions in this method - but yeah I would also prefer the `"%" in netloc` test.
should I change it also some lines below (_1197_) ?

Reply:
yes, please use "in" in your whole PR.

Reply:
ok, will change it as soon as I'm back on Sunday.

Reply:
@vstinner I've adjusted this PR to use the "in" operator in the if statements